src/tests/integration/conftest.py

```diff
@@ -36,6 +42,11 @@ def custom_temp_dir(request: pytest.FixtureRequest) -> Path:
     return request.getfixturevalue('tmp_path')
 
 
+@pytest.fixture
+def enable_bug_report(request: pytest.FixtureRequest) -> bool:
```

Comment:
```suggestion
def save_bug_report(request: pytest.FixtureRequest) -> bool:
```

Or, rename the flag to `--enable-bug-report`.